index.ts

```diff
@@ -30,7 +70,7 @@ const searchUrl = `${mapzenUrl}/search`
  * @param {string} [$0.url='https://search.mapzen.com/v1/autocomplete']                       optional URL to override Mapzen autocomplete endpoint
  * @return {Promise}                              A Promise that'll get resolved with the autocomplete result
  */
-export function autocomplete ({
+export const autocomplete = ({
```

Comment:
Generally I do prefer the `export function autocomplete (...) { }` syntax.

Reply:
Changed in 9cef1b0e792b09793659b2070713873414974c77!